docs/cri/config.md

```diff
@@ -168,6 +168,11 @@ version = 2
       # i.e pass host devices through to privileged containers.
       privileged_without_host_devices = false
 
+      # privileged_without_host_devices_all_devices_allowed allows the whitelisting of all devices
+      # when privileged_without_host_devices is enabled. Requires privileged_without_host_devices to
+      # be enabled. Defaults to false.
+      privileged_without_host_devices_all_devices_allowed = false
```

Comment:
How is this different from the plain `privileged`?

Reply:
In plain privilege all host device nodes are added to the container's spec and all devices are whitelisted with `oci.WithHostDevices` and `oci.WithAllDevicesAllowed`. This flags is for the modification of the `privileged_without_host_devices` so that even when no host devices are implicitly added to the container, `oci.WithAllDevicesAllowed` is still enabled.

Reply:
Could you put the explanation to `config.md`?

Reply:
your explanation:
```
In plain privilege all host device nodes are added to the container's spec and all devices are whitelisted with oci.WithHostDevices and oci.WithAllDevicesAllowed. This flag is for the modification of the privileged_without_host_devices so that even when no host devices are implicitly added to the container, oci.WithAllDevicesAllowed is still enabled.
```
is easier to understand than the text in your commit to the config.md file where people will read about this..
```
      # privileged_without_host_devices_all_devices_allowed allows the allowlisting of all devices
      # when privileged_without_host_devices is enabled. This is useful for when the behaviour on
      # non-implicit mounting of host devices node is needed but all devices allowlisting behaviour of plain
      # privilege is needed. Requires privileged_without_host_devices to be enabled. Defaults to false.
```

Reply:
The new text has been updated to 
```
      # privileged_without_host_devices_all_devices_allowed allows the allowlisting of all devices when
      # privileged_without_host_devices is enabled.
      # In plain privileged mode all host device nodes are added to the container's spec and all devices
      # are put in the container's device allowlist. This flags is for the modification of the privileged_without_host_devices
      # option so that even when no host devices are implicitly added to the container, all devices allowlisting is still enabled.
      # Requires privileged_without_host_devices to be enabled. Defaults to false.
```